### ISSUE TITLE
Tests fail on Ember 2.10++

### ISSUE THREAD

**Issue by blimmer** [minutebase/ember-can#50]
```
not ok 12 PhantomJS 1.9 - Acceptance | IfCanTest: changing the resource should update the helper contents
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at http://localhost:7357/assets/test-support.js:3881
                at http://localhost:7357/assets/test-support.js:4611
                at http://localhost:7357/assets/vendor.js:47885
                at adapterDispatch (http://localhost:7357/assets/vendor.js:49020)
                at dispatchError (http://localhost:7357/assets/vendor.js:27441)
                at onerrorDefault (http://localhost:7357/assets/vendor.js:40944)
                at http://localhost:7357/assets/vendor.js:68388
                at http://localhost:7357/assets/vendor.js:69272
                at http://localhost:7357/assets/vendor.js:10894
                at http://localhost:7357/assets/vendor.js:10962
                at http://localhost:7357/assets/vendor.js:11086
                at http://localhost:7357/assets/vendor.js:11156
                at http://localhost:7357/assets/vendor.js:11279
                at run (http://localhost:7357/assets/vendor.js:32069)
                at http://localhost:7357/assets/vendor.js:48780
        message: >
            TypeError: 'undefined' is not a function (evaluating 'this.get('helper').compute(params, hash)')
        Log: |
            { type: 'error',
              text: '\'TypeError: \\\'undefined\\\' is not a function (evaluating \\\'this.get(\\\'helper\\\').compute(params, hash)\\\')\\n    at compute 
```

https://travis-ci.org/minutebase/ember-can/builds/193584199

Related to glimmer2?

**Comment by GabKlein** [comment]
Yes, I'm using ember 2.11 and ember-can is not working.

```js
this.get('helper').compute
undefined

this.get('helper')
qonto@helper:can:
```

**Comment by vluoto** [comment]
Changing the line [here](https://github.com/minutebase/ember-can/blob/v0.8.2/addon/helpers/cannot.js#L7) from

```js
return getOwner(this).lookup('helper:can');
```

to

```js
return getOwner(this).lookup('helper:can').create();
```

seems to fix it

**Comment by vluoto** [comment]
btw @blimmer this ought to be fixed by #52

**Comment by blimmer** [comment]
yep - thanks - closing!
